README.md

```diff
@@ -119,6 +119,7 @@ If you clone this repository, use the checkout commands to access the sources fo
 | [Cockpit Sample Plugin](/cockpit/cockpit-sample-plugin)                    | Plugin                    |
 | [Failed Jobs Plugin](/cockpit/cockpit-failed-jobs-plugin)                  | Plugin                    |
 | [Javascript Only Plugin](/cockpit/js-only-plugin)                          | Plugin, Custom Script     |
+| [JQuery 3.4 Behavior Patch](/cockpit/jquery-34-behaviour)                  | Plugin, Custom Script     |
```

Comment:
Can we use one or the other spelling of behavior but not mix it? I would prefer the US edition since this is the one that is predominantly used at Camunda.